src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: 300ms;
```

Comment:
1. Move transition time to variables, you use it several times.
2. Specify which exact properties are you transitioning.